util/applications/src/iScience/generate_reads/infer_weights_from_hll_sketches.cpp

```diff
@@ -0,0 +1,40 @@
+// SPDX-FileCopyrightText: 2006-2024 Knut Reinert & Freie Universität Berlin
+// SPDX-FileCopyrightText: 2016-2024 Knut Reinert & MPI für molekulare Genetik
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <hibf/sketch/compute_sketches.hpp>
+#include <hibf/sketch/hyperloglog.hpp>
+
+#include <raptor/file_reader.hpp>
+
+#include "cmd_arguments.hpp"
+
+namespace raptor::util::generate_reads
+{
+
+void infer_weights_from_hll_sketches(cmd_arguments & arguments)
+{
+    auto input_fn = [&arguments](size_t const ub_id, seqan::hibf::insert_iterator it)
+    {
+        static raptor::file_reader<raptor::file_types::sequence> const reader{seqan3::ungapped{32u}, 32u};
```

Comment:
should this be fixed to 32? 
hm thinking about it I think its fine. 

But can you put it into the `cmd_arguments` (internal State) s.t. all configuration parameters are at one place?

Reply:
I don't think it needs to be configurable. I'd argue that the highest possible value yields the best weight.